Production/python/get_py.py

```diff
@@ -2,7 +2,7 @@
 # check os
 slc = os.environ['SCRAM_ARCH'].split('_')[0]
 extra_paths = [
-    "/cvmfs/cms.cern.ch/crab3/"+slc+"_amd64_gcc493/cms/dbs3-client/3.5.1/lib/python2.7/site-packages/",
+    "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf/lib/",
     "/cvmfs/cms.cern.ch/crab3/"+slc+"_amd64_gcc493/cms/dbs3-pycurl-client/3.5.1/lib/python2.7/site-packages/",

```

Comment:
this should probably just be removed

Reply:
The pycurl path has been removed. I tested `get_py.py` and it still works.